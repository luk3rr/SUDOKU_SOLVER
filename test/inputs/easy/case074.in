000810090 804006000 000020007 060708000 080031024 003040000 140000000 005004800 000390001 
