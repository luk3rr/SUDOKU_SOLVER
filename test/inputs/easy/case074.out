Solving the following grid:
0 0 0 | 8 1 0 | 0 9 0 
8 0 4 | 0 0 6 | 0 0 0 
0 0 0 | 0 2 0 | 0 0 7 
------+-------+------
0 6 0 | 7 0 8 | 0 0 0 
0 8 0 | 0 3 1 | 0 2 4 
0 0 3 | 0 4 0 | 0 0 0 
------+-------+------
1 4 0 | 0 0 0 | 0 0 0 
0 0 5 | 0 0 4 | 8 0 0 
0 0 0 | 3 9 0 | 0 0 1 


Solution found :')

7 2 6 | 8 1 3 | 4 9 5 
8 9 4 | 5 7 6 | 1 3 2 
3 5 1 | 4 2 9 | 6 8 7 
------+-------+------
4 6 2 | 7 5 8 | 9 1 3 
9 8 7 | 6 3 1 | 5 2 4 
5 1 3 | 9 4 2 | 7 6 8 
------+-------+------
1 4 9 | 2 8 7 | 3 5 6 
2 3 5 | 1 6 4 | 8 7 9 
6 7 8 | 3 9 5 | 2 4 1 

Algorithm: BFS
Total time: 121 ms
Total expanded states: 20368
